config/initializers/yabeda.rb

```diff
@@ -0,0 +1,12 @@
+if ENV.key?("VCAP_SERVICES")
+  vcap_config = JSON.parse(ENV["VCAP_APPLICATION"])
+  app_name = vcap_config["name"]
+  app_guid = vcap_config["application_id"]
+  org_name = vcap_config["organization_name"]
+  space_name = vcap_config["space_name"]
+  app_instance = ENV["CF_INSTANCE_INDEX"]
+
+  Yabeda.configure do
+    default_tag :app, app_name, :exported_instance app_guid, :organisation: org_name, :space space_name
```

Comment:
does `:organisation:` work? should the ones that are `:symbol value` be `symbol: value` or `:symbol, value`?